### PR TITLE
Added missing Discovery proxy feature instructions

### PR DESCRIPTION
**Pull request by lzap**
@sghai we missed this information. I will add similar instructions to the
downstream document as well.

### CODE REVIEW

**Comment by sghai** [comment]
Cool..thanks for raising this..

File: plugins/foreman_discovery/2.0/index.md [hunk 1]
```diff
@@ -341,7 +341,18 @@ This should by done automatically by our installer. It is a good idea to check
 if the host responds properly and there are no firewalls blocking the
 communication.
 
-## 3.3 Permissions
+## 3.3 Subnet proxy setup
+
+When using Foreman Proxy Discovery plugin, it is required to specify subnets
+for which proxy will be used for communication towards discovered nodes.
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: `for which the proxy`

File: plugins/foreman_discovery/2.0/index.md [hunk 2]
```diff
@@ -341,7 +341,17 @@ This should by done automatically by our installer. It is a good idea to check
 if the host responds properly and there are no firewalls blocking the
 communication.
 
-## 3.3 Permissions
+## 3.3 Subnet proxy setup
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: 3.2.1 or .2 perhaps?

File: plugins/foreman_discovery/2.0/index.md [hunk 3]
```diff
@@ -341,7 +341,17 @@ This should by done automatically by our installer. It is a good idea to check
...
+
+All subnets containing discovered nodes need to be specified to allow Foreman
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: Ah, context always helps.  Let me suggest a more accurate rewrite:

"All subnets with discovered nodes need this specified in Foreman so it connects via the Foreman Proxy."

File: plugins/foreman_discovery/2.0/index.md [hunk 4]
```diff
@@ -341,7 +341,17 @@ This should by done automatically by our installer. It is a good idea to check
...
+application to connect to them. To do this, go to *Infrastructure > Smart
+Proxies* and verify if the desired proxy lists Discovery feature. If not,
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: `lists the *Discovery* feature`

File: plugins/foreman_discovery/2.0/index.md [hunk 5]
```diff
@@ -341,7 +341,17 @@ This should by done automatically by our installer. It is a good idea to check
...
+click on Refresh features button and it will appear immediately.
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: `click on the *Refresh features* button`

File: plugins/foreman_discovery/2.0/index.md [hunk 6]
```diff
@@ -341,7 +341,17 @@ This should by done automatically by our installer. It is a good idea to check
...
+
+In *Infrastructure > Subnets* select desired Smart Proxy for each appropriate
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: `select the desired *Discovery Proxy* for each`

**Comment by lzap** [comment]
All done. Thanks.

File: plugins/foreman_discovery/2.0/index.md [hunk 7]
```diff
@@ -341,6 +341,16 @@ This should by done automatically by our installer. It is a good idea to check
...
 communication.
 
+## 3.2.1 Subnet proxy setup
+
+All subnets with discovered nodes need this specified in Foreman so it
+connects via the Foreman Proxy. To do this, go to *Infrastructure > Smart
+Proxies* and verify if the desired proxy lists *Discovery* feature. If not,
+click on *Refresh features* button and it will appear immediately.
+
+In *Infrastructure > Subnets* select desired *Discovery Proxy* for each
```
Comment (domcleal) [inline plugins/foreman_discovery/2.0/index.md]: `select the desired`

Comment (lzap) [inline plugins/foreman_discovery/2.0/index.md]: Done

## 

Later,
 Lukas #lzap Zapletal

**Comment by domcleal** [comment]
Thanks!

**Pull request merged**